runtime/jvmti/jvmtiThread.cpp

```diff
@@ -472,8 +489,30 @@ jvmtiInterruptThread(jvmtiEnv *env,
 			releaseVMThread(currentThread, targetThread, thread);
 		}
 done:
-		vm->internalVMFunctions->internalExitVMToJNI(currentThread);
+		vmFuncs->internalExitVMToJNI(currentThread);
+
+#if JAVA_SPEC_VERSION >= 19
+		if ((JVMTI_ERROR_NONE == rc) && isVirtualThread) {
+			if (NULL == vm->vThreadInterrupt) {
+				jmethodID vThreadInterrupt = jniEnv->GetMethodID(vThreadJClass, "interrupt", "()V");
+
+				if (NULL == vThreadInterrupt) {
+					rc = JVMTI_ERROR_INTERNAL;
+					goto exit;
+				}
+
+				vm->vThreadInterrupt = vThreadInterrupt;
+			}
+
+			jniEnv->CallObjectMethod(thread, vm->vThreadInterrupt);
+
+			if (jniEnv->ExceptionOccurred()) {
+				rc = JVMTI_ERROR_INTERNAL;
+			}
+		}
 	}
+exit:
+#endif /* JAVA_SPEC_VERSION >= 19 */
```

Comment:
I'm pretty sure this is causing builds for Java < 19 to fail. The `#endif` extends past the brace now.

Reply:
I agree. I'll create a PR to fix that.

Reply:
See https://github.com/eclipse-openj9/openj9/pull/18971.